src/Superpower/Parsers/Span.cs

```diff
@@ -258,5 +258,39 @@ public static TextParser<TextSpan> MatchedBy<T>(TextParser<T> parser)
                     result.Remainder);
             };
         }
+
+        /// <summary>
+        /// Parse as much of the input as matches <paramref name="text" />.
```

Comment:
_Matches text from the source string until <paramref name="text" /> is seen, or the end of input is reached._ ? (See below for the variation regarding EOF.)